scripts/setup-helper-functions.sh

```diff
@@ -141,6 +141,11 @@ function cmake_install {
     -DBUILD_TESTING=OFF \
     "$@"
   ninja -C "${BINARY_DIR}"
-  sudo ninja -C "${BINARY_DIR}" install
+  local OS=`uname -s`
+  if [ $OS == 'Darwin' ]; then
+    ninja -C "${BINARY_DIR}" install
+  else
+    sudo ninja -C "${BINARY_DIR}" install
```

Comment:
I'm not sure why we need `sudo`. It seems vanilla velox do not have it. I do not know the history so I do not change it in this pr.